src/pyramid_debugtoolbar/panels/templates/session.dbtmako

```diff
@@ -0,0 +1,90 @@
+<h5>Session Configuration</h5>
+% if configuration:
+	<p>Using `ISessionFactory`: <code>${configuration}</code></p>
+% else:
+	<p>No `ISessionFactory` Configured</p>
+% endif
+
+
+## Panel Active
+% if is_active:
+	<p>This panel <span class="label label-success">was active</span> for this
+	request, and will always display the "Session" data -- even if `.session`
+	was not accessed during the request.
+	This behavior can be toggled with the Panel Settings.</p>
```

Comment:
Move this outside the `if/else` and put into a separate paragraph to avoid repeating yourself and to make it more prominent.

Also I am not clear where one would access Panel Settings. Do you mean Settings > Panel Settings, in config, both, or something else?

Reply:
I've changed the text to read: "This behavior can be toggled via the Toolbar Settings tab."

The phrase "Toolbar Settings tab" has been used elsewhere in the docs.